### DISCUSSION TITLE
No Valid Chat Context

### DISCUSSION THREAD

**Discussion by aagha** [logancyang/obsidian-copilot#327] [Q&A]
I've set up LM STudio and downloaded the Zephyr model and started the server.

When I go to the plugin in Obsidian, any query I run shows the same error:

![image](https://github.com/logancyang/obsidian-copilot/assets/5522103/c0996455-4266-475f-ba4e-878cbe0707c9)

I have setup the plugin to use the local server:

![image](https://github.com/logancyang/obsidian-copilot/assets/5522103/54af9d4c-db9a-4b3d-8cdc-4ea0026db8d0)
![image](https://github.com/logancyang/obsidian-copilot/assets/5522103/f453fc47-94e7-4e61-950e-41c6dd852ee5)

Do any steps need to be taken to make the model aware of my vault/location/folders?

**Comment by logancyang** [comment]
Did you turn on CORS? Please check the local copilot setup guide.

**Reply by aagha** [reply]
I literally thought I verified that, but that was it. Sorry.
